docs/en/stack/ml/df-analytics/ml-supervised-workflow.asciidoc

```diff
@@ -0,0 +1,139 @@
+{classification}.
+
+[[train-test-iterate]]
+== Train, test, iterate
+
+After your data is prepared and transformed into the right format, it is time to 
+train the model. Training is an iterative process — every iteration is followed 
+by an evaluation to see how the model performs. When you are satisfied with the 
+results, you are ready to deploy the model otherwise you may want to adjust the 
+training configuration or consider alternative ways to preprocess and represent 
+your data.
+
+The first step is defining the features – the relevant fields in the data set – 
+that will be used for training the model. By default, all the fields with 
+supported types are included in {regression} and {classification} automatically. 
+However, you can exclude irrelevant fields optionally from the process. Doing so 
+makes a large dataset more manageable, reducing the computing resources and time 
+required for training.
+
+Now define how to split your data into a training and a test set. The test set 
+won’t be used to train the model, but is used to evaluate how the model 
+performs. There is no optimal percentage that fits all use cases, it depends on 
+the amount of data and the time you have to train. For large data sets, you may 
+want to start with a low training percent to complete an end-to-end iteration in 
+a short time.
+
+During the training process, the training data that contains the ground truth 
+you want to learn from is fed through the learning algorithm. The model predicts 
+the value and compares it to the ground truth then the model is fine-tuned to 
+make the predictions more accurate.
+
+Once the model has been trained, it will be tested on how well it predicts for 
+previously unseen data. This test provides an estimate of a quantity known as 
+the model generalization error. There are further 
+<<ml-dfanalytics-evaluate,evaluation types>> for both {regression} and 
+{classification} analysis which provide metrics about training performance. 
+Depending on the evaluation results, you may wish to iterate.
+
+
+[[deploy-model]]
+== Deploy model
+
+You have trained the model and are satisfied with the performance. The last step 
+is to deploy your trained model and start using it on new data.
+
+The Elastic {ml} feature called {infer} enables you to make predictions for new 
+data either by using it as a processor in an ingest pipeline, in a continuous 
+{transform} or as an aggregation at search time. When new data comes into your 
+ingest pipeline or you run a search on your data with an {infer} aggregation, 
+the model is used to infer against the data and make predictions on it.
+
+
+[[next-steps]]
+== Next steps
+
+* Read more about how to {ref}/transforms.html[transform you data] into an 
+  entity-centric index.
+* Consult the documentation to learn more about <<dfa-regression,regression>> 
+  and <<dfa-classification,classification>>.
+* Learn how to evaluate <<ml-dfanalytics-regression-evaluation,regression>> and 
```

Comment:
I thought these were the same links as the previous sentence, since they have the same name. I suggest linking the "evaluate" term instead:
```suggestion
* Learn how to <<ml-dfanalytics-evaluate,evaluate>> regression and 
```